workers/unity/Assets/Fps/Scripts/SetupLogic/AndroidWorkerConnector.cs

```diff
@@ -23,6 +22,7 @@ public class AndroidWorkerConnector : MobileWorkerConnector, ITileProvider
         private const string Small = "small";
         private const string Large = "large";
 
+        public bool ShouldConnectLocally;
```

Comment:
considering to move this into the MobileWorkerConnector as it is difficult to determine whether the app should connect locally or to a cloud deployment without relying on command line arguments